examples/src/bin/dynamic_output_gpio.rs

```diff
@@ -0,0 +1,70 @@
+//! Iterates over an array of LEDs, demonstrating how to use AnyPin and Flex to drive GPIOs.
+//!
+//! The following wiring is assumed:
+//! - LED => GPIO0
+//! - LED => GPIO1
+//! - LED => GPIO2
+
+//% CHIPS: esp32c6
+
+#![no_std]
+#![no_main]
+
+use core::primitive::u32;
```

Comment:
Why is this necessary?

Reply:
I wanted to make a fancy random blinky out of that, but I decided to make it simpler instead.
Well idk how rust analyzer didn't catch this, but it's gone now